db/queries/reporting/queries.py

```diff
@@ -102,19 +109,32 @@ def get_report_for_applications(
     return_json_list = []
     for application in applications:
         return_json = {field: None for field in get_key_report_field_headers()}
-        for form in application["forms"]:
-            if form.get("name") in [
-                form.get("form_name") for form in KEY_REPORT_MAPPING
-            ]:
-                for question in form["questions"]:
+        if application["language"] == "en":
+            report_config_forms = [
+                report_config.get("form_name")
+                for report_config in KEY_REPORT_MAPPING
+            ]
+        elif application["language"] == "cy":
+            report_config_forms = [
+                report_config.get("form_name_cy")
+                for report_config in KEY_REPORT_MAPPING
```

Comment:
Could we do this in one assignment/statement e.g.
```
report_config.get("form_name_cy") if application["language"] == "cy" else report_config.get("form_name")
```
(untested but hopefully gives an idea)

Reply:
This worked thanks @gidsg 
`        report_config_forms = [report_config.get("form_name_cy") if application['language'] == 'cy' else report_config.get("form_name") for report_config in KEY_REPORT_MAPPING]
`